src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -215,4 +231,41 @@ public void immutableSources() {
 
     Assertion.assertEquals("AutomatedTest", invocationLabelText);
   }
+
+
+  //TODO: add cases for clicking 'yes' and 'no' buttons
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1135)